src/models/Act.ts

```diff
@@ -0,0 +1,6 @@
+export const Act: Object = {
```

Comment:
Use `export interface Act` or `export Class Act` I'll leave discovery and differentiation of the two to you to identify.
Here's a quick article to get started, https://ultimatecourses.com/blog/classes-vs-interfaces-in-typescript

Reply:
Apologies. I'd had actually already updated this but did not push it with the correct changes.